modules/enterprise/server/installer/src/main/java/org/rhq/enterprise/server/installer/ServerInstallUtil.java

```diff
@@ -241,11 +241,12 @@ public static void configureTransactionManager(ModelControllerClient mcc, HashMa
         client.setDefaultTransactionTimeout(600);
         LOG.info("Default transaction timeout set to 600 seconds.");
 
-        if(serverProperties.containsKey(ServerProperties.PROP_JBOSS_HA_NODE_ID)) {
-            String nodeId = buildExpression(ServerProperties.PROP_JBOSS_HA_NODE_ID, serverProperties, true, false, false);
-            client.setTransactionNodeId(nodeId);
-            LOG.info("Set high availability nodeId to " + nodeId);
-        }
+        // Set expression and its default regardless if a value was specified at install time
+        final HashMap<String, String> transactionNodeIdDefault = new HashMap();
+        transactionNodeIdDefault.put(ServerProperties.PROP_JBOSS_HA_NODE_ID, "1");
+        String nodeId = buildExpression(ServerProperties.PROP_JBOSS_HA_NODE_ID, transactionNodeIdDefault, true, false, false);
+        client.setTransactionNodeId(nodeId);
+        LOG.info("Set high availability nodeId to " + nodeId);
```

Comment:
I think this logging should be removed if we're not setting anything that's been defined. It could be confusing otherwise-

Reply:
+1

Reply:
Removed.